lib/system/zephyr/sys.h

```diff
@@ -31,7 +31,7 @@ extern "C" {
 }
 
 #ifndef METAL_MAX_DEVICE_REGIONS
-#define METAL_MAX_DEVICE_REGIONS 1
+#define METAL_MAX_DEVICE_REGIONS 2
```

Comment:
This should be defined in zephyr. in prj.conf or in Cmakelists.txt
for instance: https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/ipc/openamp_rsc_table/CMakeLists.txt#L15